scipy/linalg/tests/test_special_matrices.py

```diff
@@ -640,3 +640,32 @@ def test_fiedler_companion():
     fc = fiedler_companion([1., -16., 86., -176., 105.])
     assert_array_almost_equal(eigvals(fc),
                               np.array([7., 5., 3., 1.]))
+
+def test_convmtx():
+    'test convmtx matrix multiply against numpy.convolve for various parameters'
+
+    def gaussian(n, cpx):
```

Comment:
Why do you need specifically gaussian distribution here? Any random vector should work here right? Also test files also better have PEP8 compatibility.

Reply:
Does not need to be Gaussian, per se.  I've updated the names to clarify this.  I added a double line above the function def.  If you are referring to something else that is not PEP8, I'm missing it (as does `autopep8`)

Reply:
I meant that the line is longer than 79 chars.

Reply:
fixed